### PR TITLE
Lps 156865 2

### PR DESCRIPTION
**Pull request by JavierMoral**
Hi team,
I applied the changes proposed in #1788 . If there are more changes to apply please tell me.

### CODE REVIEW

**Comment by JavierMoral** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:test:upgrade

**Comment by rodrigopaulino** [comment]
ci:help

**Comment by rodrigopaulino** [comment]
ci:test:upgrade

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:stop

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			while (resultSet.next()) {
+				long companyId = resultSet.getLong(1);
+
+				long primKey = resultSet.getLong(2);
+
+				Role guestRole = _roleLocalService.getRole(
+					companyId, RoleConstants.GUEST);
+
+				ResourcePermission resourcePermission =
+					_resourcePermissionLocalService.getResourcePermission(
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java:58]: You don't need to check if the resource permission has the action ID. The method `removeResourcePermission` already does that. You can simply call `removeResourcePermission` without checking anything.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement selectPreparedStatement =
+				connection.prepareStatement(
+					"select companyid, forminstanceid from DDMFormInstance");
+			ResultSet resultSet = selectPreparedStatement.executeQuery()) {
+
+			while (resultSet.next()) {
+				long companyId = resultSet.getLong(1);
+
+				long primKey = resultSet.getLong(2);
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java:52]: When you're left with just one usage of this variable, you can get the value directly from the method call.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+				_resourcePermissionLocalService.removeResourcePermission(
+					companyId, _CLASS_NAME_FORM_INSTANCE,
+					ResourceConstants.SCOPE_INDIVIDUAL, String.valueOf(primKey),
+					guestRole.getRoleId(), ActionKeys.VIEW);
+			}
+		}
+	}
+
+	private static final String _CLASS_NAME_FORM_INSTANCE =
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_1_5/DDMFormInstanceUpgradeProcess.java:75]: Could you please change this to `_CLASS_NAME_DDM_FORM_INSTANCE`?

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hi, @JavierMoral. Thanks a lot for contributing and resending this fix. We really appreciate it.
I have requested just a few changes. You can find them below.
Thanks!

**Comment by rodrigopaulino** [comment]
Closing due to inactivity for more than 10 days. Feel free to send us a new PR with the proposed changes if the fix is still required.

**Comment by rodrigopaulino** [comment]
ci:close

**Pull request closed without merging**